src/segmantic/seg/monai_unet.py

```diff
@@ -390,6 +500,9 @@ def predict(
     test_labels: Optional[List[Path]] = None,
     output_dir: Path = None,
     tissue_dict: Dict[str, int] = None,
+    channels: tuple = (16, 32, 64, 128, 256),
+    strides: tuple = (2, 2, 2, 2),
```

Comment:
please use `Tuple[int]`, `List[str]`, etc instead of `tuple`, `list` unless the type could be any

Reply:
Changed it~~ thank you